src/share/poudriere/common.sh

```diff
@@ -7661,6 +7661,35 @@ calculate_ospart_size() {
 	msg "OS Partiton size: ${OS_SIZE}m"
 }
 
+
+# Stolen from bsdconfig
+# str_replaceall $string $find $replace [$var_to_set]
```

Comment:
We have a function for this already:
```
_gsub "$string" "$match" "$replace"
# $_gsub contains result
# or use result=$(gsub ...)
```
I just pushed this new version to match the API you used. So you should just be able to change the function call name to `_gsub`.
```
_gsub "$string" "$match" "$replace" "$output"
```